apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -3,84 +3,180 @@ defmodule Andi.InputSchemas.InputConverter do
   Used to convert between SmartCity.Datasets, form data (defined by Andi.InputSchemas.DatasetInput), and Ecto.Changesets.
   """
 
-  alias SmartCity.Dataset
-  alias Andi.InputSchemas.DatasetInput
+  alias Andi.InputSchemas.Datasets.Dataset
```

Comment:
Should the converter methods live with the schemas they convert from/to?

Reply:
I'm not sure these functions are specific enough to one of the input schemas to do that. We could probably move some of them, but it made sense to consolidate them to one place, imo. But I can leave that up to discussion